client/app/components/account_config_main.coffee

```diff
@@ -1,3 +1,4 @@
+AccountActionCreator = require '../actions/account_action_creator'
```

Comment:
We both did this fix, so the import is now twice in the file (here and at line 10)